src/components/utils/file.js

```diff
@@ -1,16 +1,23 @@
 import loadImage from 'blueimp-load-image/js/load-image'
 import {canvasToBase64Images, toLossyImageDataUrl} from './canvas.js'
 
-export const fileToBase64 = (file, callback, errorCallback) => {
+export const convertFile = (file, callback, errorCallback) => {
   const reader = new FileReader();
-  reader.readAsDataURL(file);
-  reader.onload = function () {
-   callback(reader.result)
+  // If PDF return base64. For other images return blob
```

Comment:
it seem `File` inherits from `Blob`, we can return a File or a Blob depending when it's more convenient and we will still respect the `Blob` interface.